website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md

```diff
@@ -0,0 +1,640 @@
+---
+title: "dbt Semantic Layer integration" 
+id: "sl-partner-integration-guide"
+description: Learn about partner integration guidelines, roadmap, and connectivity. 
+---
+
+# dbt Semantic Layer partner integration
+
+:::info Contact us
+
+This guide describes dbt Semantic Layer integration guidelines, product roadmap, and connectivity.  <br />
+
+To become a formal partner, integrate via the API, or if you have questions or feedback &mdash; **[contact us](mailto:semantic-layer@dbtlabs.com)** for more info.
+
+:::
+
+The dbt Semantic Layer allows users to dynamically generate and query datasets in downstream tools based on their dbt governed assets, like metrics, models, and entities. It helps organizations manage complexities like data, tools, and teams to make more efficient and trustworthy decisions.
+
+## Overview
+
+The rapid growth of different tools in the modern data stack has largely been a good thing for data professionals - the many different tools address the different needs of different teams. The downside of this growth is the fragmentation of business logic across teams, tools, and workloads. 
+
+To solve this, the dbt Semantic Layer provides a platform where users can confidently leverage their data from within their tools. dbt Cloud's change management capabilities ensure that any user modifications made to core business constructs, like metrics or entities, are distributed into all the tools connected to the data platform.
+
+The dbt Semantic Layer is unopinionated about specific tools, use cases, or applications of data. The following summarizes some of the common use cases for the dbt Semantic Layer:
+
+* BI, reporting, and analytics
+* Data quality and monitoring
+* Governance and privacy
+* Data discovery and cataloging
+* Machine learning and data science
+
+<!-- rewrite below little and would like to see if i can include a button or callout -->
+
+:::info Share your use case
+
+To share other use cases for the dbt Semantic Layer, contact the [dbt Labs team](mailto:semantic-layer@dbtlabs.com).
+
+:::
+
+
+## Product overview
+
+The dbt Semantic Layer product architecture includes four primary components:
+
+| Components | Information | Developer plans | Team plans | Enterprise plans | License |
+| --- | --- | :---: | :---: | :---: | --- |
+| **[dbt Project](/docs/build/metrics)** | Define models and metrics in dbt Core. | ✅ | ✅ |  ✅  | Open source in dbt Core |
+| **[dbt Server](https://github.com/dbt-labs/dbt-server)**| A persisted HTTP server that wraps dbt Core to handle RESTful API requests for dbt operations. | ✅ | ✅ | ✅ | BSL |
+| **SQL Proxy** | Reverse-proxy that accepts dbt-SQL (SQL + Jinja-like query models and metrics, use macros), compiles the query into pure SQL, executes the query in the data platform, and returns the data. | ✅ <br></br>_* Available during Public Preview only_ | ✅ | ✅ | Proprietary in dbt Cloud |
```

Comment:
do we need to add any link to the sql proxy ?

```suggestion
| **SQL Proxy** | Reverse-proxy that accepts dbt-SQL (SQL + Jinja-like query models and metrics, use macros), compiles the query into pure SQL, executes the query in the data platform, and returns the data. | ✅ <br></br>_* Available during Public Preview only_ | ✅ | ✅ | Proprietary in dbt Cloud |
```

Reply:
No, there isn't additional info about it